### DISCUSSION TITLE
Edge 浏览器 UI 显示 Bug

### DISCUSSION THREAD

**Discussion by DDDalek92** [1Panel-dev/1Panel#6397] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

数据库，容器，主机文件等涉及列表显示的页面，在右侧操作列，UI显示错乱，其中主机文件页面顶部的前进后退等导航按钮也显示错乱

### 重现步骤

清理浏览器缓存后，UI任然显示错乱

### 附加信息

![0f2ace4ebd61479bf42606409fdd4a8](https://github.com/user-attachments/assets/d00c5918-af85-4db7-b0b3-3eb213eb6ea2)

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。上述问题可能是浏览器适配问题，你可以先使用 Chrome 或 Firefox 浏览器访问 1Panel 控制台。我们将在后续版本中优化对 Edge 浏览器的支持。

**Comment by wan92hen** [comment]
可以检查下是不是安装了百度网盘的插件，是的话禁用掉再看下。
